openapi-v1.yaml

```diff
@@ -2729,6 +2740,28 @@ definitions:
         type: string
         description: output location of the exported file
 
+  NotebookCopy:
```

Comment:
@antalakas some questions on the design? 
- what is output_uri? Can it be a general uri? Why we don't just copy to `default_storage_location/notebooks/new_name`
- why namespace is missing? An admin may want to copy a notebook for an organization

In general what are the drawbacks of just using `namespace, name`?

Reply:
`namespace` is added, let's stick with absolute `output_uri` and make the discussion regarding abstractions in later PRs when reviewing onboarding and user experience enhancements